.github/workflows/gh-pages.yml

```diff
@@ -29,10 +29,12 @@ jobs:
           python -m pip install -r requirements.txt -r requirements-dev.txt
           python -m pip install '.[all]'
       - uses: actions/cache@v4
+        id: cache-docs
         with:
           path: docs/_build
           key: ${{ runner.os }}-sphinx
-      - name: Generate the documentation
+      - if: ${{ steps.cache-docs.outputs.cache-hit != 'true' }}
+        name: Generate the documentation
         run: |
           cd docs/
           make html
```

Comment:
This is not working as intended. These two were skipped in the [latest run](https://github.com/tqec/tqec/actions/runs/15452216197/job/43496802675?pr=592)

![image](https://github.com/user-attachments/assets/71a64e6c-3045-4cb8-9366-eb026d2bfbb5)